### PR TITLE
Fix #2712: Log translation creation action for pretranslations

### LINKED ISSUE #2712: Log translation creation action for pretranslations
We currently do not log translation creation actions for pretranslations.

The patch should also include a data migration to create missing objects for existing pretranslations.

### PR DESCRIPTION
**Pull request by vishalol**
Fixes #2712

### CODE REVIEW

**Review by mathjazz — COMMENTED** [review]
Nice work! Left some comments. Please fix the logic in migrations and we're good.

File: pontoon/actionlog/migrations/0003_existing_pretranslation_action.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.15 on 2023-01-24 10:40
+
+from django.db import migrations
+
+from pontoon.actionlog.models import ActionLog
+
+
+def create_action_logs_for_pretranslations(apps, schema_editor):
+    Translation = apps.get_model("base", "Translation")
+    ActionLogModel = apps.get_model("actionlog", "ActionLog")
```
[the remark this answers is not in the document]
Comment (vishalol) [reply pontoon/actionlog/migrations/0003_existing_pretranslation_action.py:10]: ActionLog was  colliding with ActionLog import. Should I rename the import instead ?

[the remark this answers is not in the document]
Comment (mathjazz) [reply pontoon/actionlog/migrations/0003_existing_pretranslation_action.py:10]: Missed that. You don't need to import `ActionLog`. In fact, you should [never import models directly in migrations](https://docs.djangoproject.com/en/4.1/topics/migrations/#historical-models).

**Review by mathjazz — APPROVED** [review]
Great work!

**Pull request merged**